sdk/storage/storage-file-datalake/src/clients.ts

```diff
@@ -933,10 +934,10 @@ export class DataLakeFileClient extends DataLakePathClient {
    * blobClientInternal provided by @azure/storage-blob package.
    *
    * @private
-   * @type {BlobClient}
+   * @type {BlockBlobClient}
    * @memberof DataLakeFileClient
    */
-  private blobClientInternal: BlobClient;
+  private blobClientInternal: BlockBlobClient;
```

Comment:
>BlockBlobClient [](start = 30, length = 15)

Add a new BlockBlobClientInternal instead of removing this.

Datalake will add appendblock support in the future.

Reply:
Renamed.